buildnotifylib/config.py

```diff
@@ -8,14 +8,15 @@
 
 class Preferences(object):
     def __init__(self, urls, interval, custom_script_text, custom_script_checked,
-                 sort_by_build_time, sort_by_name, selections):
+                 sort_by_build_time, sort_by_name, selections, show_last_build_label):
```

Comment:
this change also requires a change in `test/integration/config_test.py`  see +/- for change below

```
def test_should_update_preferences(self):
-        self.config.update_preferences(Preferences(['url1'], 300, '/bin/sh', True, False, True, []))
+        self.config.update_preferences(Preferences(['url1'], 300, '/bin/sh', True, False, True, [], False))

         self.assertEqual(self.config.get_urls(), ['url1'])
         self.assertEqual(self.config.get_interval_in_seconds(), 300)
```

Reply:
Nice catch!
Just fixed it :)